integreat_cms/cms/templates/pois/poi_form.html

```diff
@@ -46,67 +46,82 @@ <h1 class="heading overflow-hidden text-ellipsis">
+                            {% render_field poi_translation_form.content class+="tinymce_textarea flex-auto" %}
                         </div>
-                        {% render_field poi_translation_form.title|add_error_class:"border-red-500" class+="mb-2" %}
-                        {% if language != request.region.default_language %}
-                            <div class="bg-yellow-100 border-l-4 border-yellow-500 text-yellow-600 px-4 py-1 mb-2">
-                                {% blocktrans %}The name is currently displayed to users only in the default language{% endblocktrans %}
+                    </div>
+                    <div class="col-span-2 3xl:col-span-1 flex flex-wrap flex-col">
+                        <div class="rounded border border-solid border-blue-500 shadow-2xl bg-white">
+                            <div class="rounded p-4 bg-water-500">
+                                <h3 class="heading font-bold text-black">{% trans "SEO section" %}</h3>
                             </div>
-                        {% endif %}
-                        <div id="link-container" class="flex items-center">
-                            <label for="{{ poi_translation_form.slug.id_for_label }}" class="mr-2">{{ poi_translation_form.slug.label }}:</label>
-                            <a id="slug-link"
-                               href="{{ url_link }}{{ poi_translation_form.instance.slug }}/"
-                               class="text-blue-500 hover:underline">{{ url_link }}{{ poi_translation_form.instance.slug }}</a>
-                            <a id="edit-slug-btn" title="{% trans "Edit" %}" class="mx-2 btn-icon">
-                                <i icon-name="edit-3"></i>
-                            </a>
-                            <a id="copy-slug-btn"
-                               title="{% trans "Copy to clipboard" %}"
-                               class="btn-icon">
-                                <i icon-name="copy"></i>
-                            </a>
-                            <div class="slug-field hidden">
-                                <label for="{{ poi_translation_form.slug.id_for_label }}">{{ url_link }}</label>
-                                {% render_field poi_translation_form.slug|add_error_class:"slug-error" %}
+                            <div class="px-4 pb-4 divide-y divide-gray-200 space-y-2">
+                                <div>
+                                    <label for="{{ poi_translation_form.meta_description.id_for_label }}"
+                                           class="secondary">
+                                        {{ poi_translation_form.meta_description.label }}
+                                    </label>
+                                    <div class="help-text">{{ poi_translation_form.meta_description.help_text }}</div>
+                                    {% render_field poi_translation_form.meta_description|add_error_class:"border-red-500" %}
```

Comment:
I guess I would prefer to put this into a region setting which is false by default. Otherwise all users will still see this and might be irritated

Reply:
@ulliholtgrave what do you mean? A setting like `seo_enabled` or similar?
Alternatively, we could only show it for expert users...

Reply:
Exactly. The service team is not really happy about adding new UI elements for the users on the screen. As SEO optimization is a very big extra mile I would suggest to activate it on a region level and only activate the whole area if a region is willing to go it. Otherwise we use a lot of space on our forms with nobody using this function at all.

Reply:
* Added "Activate SEO" setting to region form
* Added condition to POI form: display SEO section if request.region.seo_enabled